examples/CustomDataframe/frontend/src/streamlit/streamlit.ts

```diff
@@ -115,10 +115,10 @@ export class Streamlit {
    * value = st.my_component(...)
    * st.write(value) # -> "ahoy!"
    *
-   * The value must be serializable into JSON.
+   * The value must be serializable into JSON or pandas.DataFrame.
    */
-  public static setComponentValue = (value: any): void => {
-    Streamlit.sendBackMsg(ComponentMessageType.SET_COMPONENT_VALUE, { value })
+  public static setComponentValue = (value: any, dataType = "json"): void => {
+    Streamlit.sendBackMsg(ComponentMessageType.SET_COMPONENT_VALUE, { value, dataType })
```

Comment:
Why is the `dataType` param required? Is it possible to introspect the value to see if its type is `ArrowTableProto` or `ArrowTable` instead?